frontend/src/scenes/activity/explore/EventDetails.tsx

```diff
@@ -145,6 +147,14 @@ export function EventDetails({ event, tableProps }: EventDetailsProps): JSX.Elem
             label: 'Conversation',
             content: (
                 <div className="mx-3 -mt-2 mb-2 space-y-2">
+                    {event.properties.$session_id ? (
```

Comment:
if you know it's a web event you can gate on recording being active

in error tracking

`issueProperties['$session_id'] && issueProperties['$recording_status'] === 'active'`

but that's not essential - we just can't guarantee that the presence of a session id means there will be a recording.